build.gradle

```diff
@@ -1,18 +1,13 @@
-apply plugin: 'application'
+plugins {
+    id 'application'
+    id 'com.google.osdetector' version '1.6.0'
```

Comment:
There is a pending issue when applying the os detector plugin this way: https://github.com/google/osdetector-gradle-plugin/issues/15

You can either fallback to 1.4.0 or configure the plugin using `buildscript`.

Reply:
@aalmiray Any update on this?

Reply:
Took care of it by reverting back to `buildscript` instead of `plugins` block